app/views/Data/Workspace/WorkTable/index.tsx

```diff
@@ -69,6 +78,53 @@ export default function WorkTable(props: Props) {
 
     const colsKeys: number[] = workspaceTable?.table?.dataColumnStats.map((col: Column) => col.key);
```

Comment:
let's not assign type this way. the type can be inferred from the `Column` type.